src/System.Private.Xml/src/System/Xml/Xsl/Runtime/XmlSortKey.cs

```diff
@@ -219,6 +219,79 @@ public override int CompareTo(object obj)
     }
 
     /// <summary>
+    /// Sort key for xs:string values.  Strings are sorted according to a byte-wise sort key calculated by caller.
+    /// </summary>
+    internal class XmlStringSortKey : XmlSortKey
```

Comment:
@stephentoub Any preferences here? Do we care about consistency with Desktop in internal classes?